pkg/controller/deployment/deployment_controller.go

```diff
@@ -124,6 +125,25 @@ func (r *ReconcileDeployment) Reconcile(request reconcile.Request) (reconcile.Re
 
 		jaeger := inject.Select(dep, ns, jaegers)
 		if jaeger != nil && jaeger.GetDeletionTimestamp() == nil {
+			if jaeger.Namespace != request.Namespace {
+				log.WithFields(log.Fields{
+					"jaeger namespace": jaeger.Namespace,
```

Comment:
For someone grepping the logs, it might be better to have `jaeger-namespace` or `jaeger.namespace` than `jaeger namespace`.